### ISSUE TITLE
Pixi on overlay click registration

### ISSUE THREAD

**Issue by hadson172** [pixijs/pixijs#8335]
Hi, i've been using pixi.js  for months now with conjunction of leaflet map. (pixi is overlay of the leaflet map to render markers, till now i've been using plugin https://github.com/manubb/Leaflet.PixiOverlay). It seems that after  upgrade leaflet map from 1.7.1 -> 1.8.0 on iphones/mobiles double click on pixi is propagated to leaflet map as single click.  Opened bug for this at leaflet.
here - https://github.com/Leaflet/Leaflet/issues/8236#issuecomment-1125299428

The people from lefleat suggested that this bug is on the site of pixi. Could you confirm or deny it? 
(BTW I used autoPreventDefault=false but it still requires two tap on iphone 8 instead of one to register clicks).

### Expected Behavior
Problem re-production available in linked issue in leaflet.

### Current Behavior
Problem re-production available in linked issue in leaflet.

### Possible Solution
Problem re-production available in linked issue in leaflet.

### Steps to Reproduce
Problem re-production available in linked issue in leaflet.

### Environment
[6.3.2](https://www.npmjs.com/package/pixi.js/v/6.3.2)
- **`pixi.js` version**: _e.g. 4.7.1_
- **Browser & Version**: Newest firefox, iOS safari, chrome
- **OS & Version**: _e.g. Ubuntu 18.04_ Windows, iOS, macOS

**Comment by IvanSanchez** [comment]
Leaflet maintainer here. From my quick analysis earlier today, it seems that Pixi is calling `preventDefault` on `touchstart` events (and other `TouchEvent`s) while ignoring `PointerEvent`s. The changes introduced in Leaflet 1.8.0 mean that Leaflet only cares about `PointerEvent`s when possible. And, for whatever reason, calling `preventDefault` on a `TouchEvent` is also preventing the corresponding `PointerEvent` from firing, at least when emulating iOS Safari.

Looking at your bug reports, this might be related to #4661.

**Comment by hadson172** [comment]
Any updates on the topic? @bigtimebuddy could you take a look on the issue ?

**Comment by bigtimebuddy** [comment]
I do not have time right now to dig into this issue. But we'd welcome a change if it could help unblock this.

It would be useful to know if this also has the same behavior with [`@pixi/events`](https://www.npmjs.com/package/@pixi/events). The events package will replace the InteractionManager in the next major release.

**Comment by Zyie** [comment]
Hi there! 👋

We're doing some housekeeping and closing out older issues that haven't had recent activity.

PixiJS has evolved significantly since this issue was opened. If you're still experiencing this issue on PixiJS v8 (latest), please open a new issue with updated details and a minimal reproduction case.

Thanks for your understanding and for helping us keep the issue tracker focused and maintainable! 🩷
